docs/tutorials/rigetti/getting_started.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>do we need to preserve output because this hits an external service to run?  (similar below)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
No, we can clear output.  Since it uses `as_qvm=True` then it is all local.

Reply:
nice